terraform-infra-approvals/global.json

```diff
@@ -31,6 +33,8 @@
     {"source":  "git@github.com:hmcts/cnp-module-action-group?ref=master"},
     {"source":  "git@github.com:hmcts/cnp-module-application-insights?ref=master"},
     {"source":  "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"},
-    {"source":  "git@github.com:hmcts/cnp-module-metric-alert?ref=master"}
+    {"source":  "git@github.com:hmcts/cnp-module-metric-alert?ref=master"},
+    {"source":  "git@github.com:hmcts/terraform-module-apim-product?ref=master"},
```

Comment:
there's already a module for this: https://github.com/hmcts/cnp-module-api-mgmt-product

Reply:
Did not know that existed. I have removed this version and will delete the repos as well

Reply:
sorry missed the add part :) done now